webapp/src/features/adminOrganizationTab/AdminOrganizationTab.tsx

```diff
@@ -101,7 +101,9 @@ const AdminOrganizationTab = () => {
   }
 
   useEffect(() => {
-    getSelectedOrgData()
+    if (selectedOrgName != undefined) {
+      getSelectedOrgData()
+    }
   }, [selectedOrgName, dispatch])
```

Comment:
suggestion: there's a few of these useEffects that are missing dependencies again. It would be good to update those (likely separate pr, but want to mention it)

Reply:
Created a backlog task: https://dev.azure.com/SOC-OIT/CDOT/_workitems/create/Product%20Backlog%20Item